include/private/PrismConnectService.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2017 Prism Skylabs
+ */
+#ifndef PRISM_PRISMCONNECTSERVICE_H
+#define PRISM_PRISMCONNECTSERVICE_H
+
+#include <client.h>
+#include <boost/shared_ptr.hpp>
+
+namespace prism
+{
+namespace connect
+{
+
+class PrismConnectService
```

Comment:
If all members/methods are public it should be a struct.
The class has no virtual methods, so no need for the destructor to be virtual.
This is not a part of the pull request, but I think it would be easier to fix it here since you are moving the file from another project.